tVip.sp

```diff
@@ -1,12 +1,13 @@
 #pragma semicolon 1
 
 #define PLUGIN_AUTHOR "Totenfluch"
-#define PLUGIN_VERSION "2.0"
+#define PLUGIN_VERSION "2.1"
 
 #include <sourcemod>
 #include <sdktools>
 #include <multicolors>
 #include <autoexecconfig>
+#include <tVip>

```

Comment:
You are not using any methods from the tVip Include so you should not include it